### DISCUSSION TITLE
Range select on DataTable

### DISCUSSION THREAD

**Discussion by Tyron199** [primefaces/community#1087] [PrimeVue] [answered]
Hi, I'm wondering if its possible select a range of rows in a datatable ? Usually when you hold shift while clicking 2 checkboxes it will also select the ones between them.

**Comment by tugcekucukoglu** [comment, marked as answer]
On the multiple selection mode holding command + shift and clicking a row selects the rows between these two rows. But the checkbox selection is different and when you want to select more than one row, it is necessary to click all the lines one by one.

**Reply by redfox05** [reply]
You've probably sorted this by now, but for you and others just in case: Note that the multiple selection depends on the flags you have set on the DataTable. Check the `selectionMode="multiple"` flag on both the table and the checkbox column. See this thread for more info: https://github.com/orgs/primefaces/discussions/2204
